app/controllers/applications_controller.rb

```diff
@@ -8,7 +8,9 @@ def create
       :email, :language_de, :language_en, :attended_before, :rejected_before, :level,
       :comments, :os, :needs_computer, :read_coc, :female))
 
-    unless @application.save
+    if @application.save
+      UserMailer.application_mail(@application).deliver_now
```

Comment:
`deliver_now` delivers the email immediately, which is great because we get instant feedback about whether it worked or not. It can be slow though, so once we see this working correctly in production, we can (in a separate PR) replace it with `deliver_later`, which does it in the background.